### PR TITLE
PBM-674 Added a FAQ section

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: doc/source/faq.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+What's the difference between PBM and ``mongodump``?
+========================================================
+
+Both |PBM| and ``mongodump`` are 'logical' backup solutions and have equal performance for non-sharded replica sets. What difference between them is that |PBM| has ``pbm-agents`` that must be installed on each ``mongod`` node. This, however, allows running backups/restores on each replica set in parallel while ``mongodump`` runs in one process on ``mongos`` node.
+
+Why does |PBM| use UTC timezone instead of server local timezone?
+==================================================================
+
+``pbm-agents`` use UTC time zone by design. The reason behind this is to avoid user misunderstandings when replica set / cluster nodes are distributed geographically in different time zones.
```
Comment (akira-kurogane) [inline doc/source/faq.rst:15]: :+1:

File: doc/source/faq.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+No, |PBM| makes backups of and restores the whole state of a replica set / sharded cluster. 
+
+If you are not using a sharded cluster, or the sharded cluster is only 2 or 3 shards, we recommend using ``-d`` and ``-c`` options with ``mongodump`` and/or ``mongorestore``. As ``mongodump``/``mongorestore`` connects directly to the primary (in a non-sharded replica set) or via a ``mongos`` node in a cluster, it sees the cluster as if it were a single node, making it simple. ``mongodump``/``mongorestore`` work in a single process, so if you aren’t reinserting to many shards, the lack of parallelization won't be too bad.
+
+Can I backup specific shards in a cluster? 
+===========================================
+
+No, since this would result in backups with inconsistent timestamps across the cluster. Such backups would be invalid for restore.
+
+PBM takes a backs up the whole state of a sharded cluster and this guarantees data consistency during the restore.
```
Comment (akira-kurogane) [inline doc/source/faq.rst:29]: :+1:

File: doc/source/faq.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+==================================================================
+
+``pbm-agents`` use UTC time zone by design. The reason behind this is to avoid user misunderstandings when replica set / cluster nodes are distributed geographically in different time zones. 
+
+Can I restore a single collection with |PBM|?
+=============================================================
+
+No, |PBM| makes backups of and restores the whole state of a replica set / sharded cluster. 
+
+If you are not using a sharded cluster, or the sharded cluster is only 2 or 3 shards, we recommend using ``-d`` and ``-c`` options with ``mongodump`` and/or ``mongorestore``. As ``mongodump``/``mongorestore`` connects directly to the primary (in a non-sharded replica set) or via a ``mongos`` node in a cluster, it sees the cluster as if it were a single node, making it simple. ``mongodump``/``mongorestore`` work in a single process, so if you aren’t reinserting to many shards, the lack of parallelization won't be too bad.
```
Comment (akira-kurogane) [inline doc/source/faq.rst:22]:
```suggestion
If single-collection restores are your primary requirement and you are not using a sharded cluster, or the sharded cluster is only 2 or 3 shards, we recommend using ``-d`` and ``-c`` options with ``mongodump`` and/or ``mongorestore``. As ``mongodump``/``mongorestore`` connects directly to the primary (in a non-sharded replica set) or via a ``mongos`` node in a cluster, it sees the cluster as if it were a single node, making it simple. ``mongodump``/``mongorestore`` work in a single process, so if you aren’t reinserting to many shards, the lack of parallelization won't be too bad.
```

**Review by akira-kurogane — CHANGES_REQUESTED** [review]

**Review by akira-kurogane — APPROVED** [review]
Thanks Anastasia!

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**